packages/web3-providers/src/types.ts

```diff
@@ -249,6 +255,8 @@ export namespace NextIDBaseAPI {
 
         queryExistedBindingByPlatform(platform: NextIDPlatform, identity: string, page?: number): Promise<any>
 
+        queryAllExistedBindingByPlatform(platform: NextIDPlatform, identity: string): Promise<NextIDPersonaBindings[]>
```

Comment:
nit: use plural form. If you accept, change `queryExistedBindingByPlatform` as well.

```suggestion
        queryAllExistedBindingsByPlatform(platform: NextIDPlatform, identity: string): Promise<NextIDPersonaBindings[]>
```